kaldb/src/main/java/com/slack/kaldb/util/JsonUtil.java

```diff
@@ -35,6 +35,12 @@ public static <T> T read(String s, TypeReference<T> valueTypeRef) throws JsonPro
     return ourInstance.mapper.readValue(s, valueTypeRef);
   }
 
+  public static <T> String writeJsonValues(String jsonString) {
```

Comment:
Can we write a small comment as to why we need this

We can call this method `writeJsonArray` since the Zipkin output format doesn't use a object but an array right?